website-next/components/projects/projects-page-component.tsx

```diff
@@ -316,6 +362,43 @@ export class ProjectsPage extends React.Component<EmptyProps, ProjectsState> {
     const visibleProjectCount =
       this.state.filteredProjects.length + this.state.filteredLocalProjects.length
 
+    const reversedProjects = [...this.state.filteredProjects].reverse()
```

Comment:
We have some utility functions in `array-utils.ts` for creating reversed arrays and for sorting by a comparator, which handle duplicating the array for you.